webpack/assets/javascripts/react_app/API.js

```diff
@@ -1,37 +1,55 @@
-import $ from 'jquery';
+import axios from 'axios';
+
+const getcsrfToken = () => {
+  const token = document.querySelector('meta[name="csrf-token"]');
+
+  if (token) {
+    return token.content;
+  }
+  // fail gracefully when no token is found
+  return '';
+};
+const onSuccess = response => response.data;
+const onError = (error) => {
+  /* eslint-disable no-console */
+  console.log(error);
```

Comment:
Do we really want console.log here?

Reply:
oh snap...  no we don't :) I forgot to delete it